mockups/pyGtkMock/guiTry.py

```diff
@@ -0,0 +1,68 @@
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk, Gdk
+from defcon import Font
+
+
+class FlowBoxWindow(Gtk.Window):
+
+    def __init__(self):
+        Gtk.Window.__init__(self, title="Character Map Demo")
+        self.set_border_width(10)
+        self.set_default_size(800, 800)
+
+        header = Gtk.HeaderBar(title="Character Map")
+        header.set_subtitle("Sample Character Map app")
+        header.props.show_close_button = True
+
+        self.set_titlebar(header)
+
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        flowbox = Gtk.FlowBox()
```

Comment:
Flowbox is the best.  But it isn't supported on the Xo's ancient Gtk version.  (We can probably hack a polyfill together if you want XO compatibility - or just get software less than 3 years old on the xo)